docs/api-reference/next/image.md

```diff
@@ -40,9 +40,29 @@ This `next/image` component uses browser native [lazy loading](https://caniuse.c
 
 ## Known Browser Bugs
 
-- [Safari 15](https://bugs.webkit.org/show_bug.cgi?id=243601) displays a gray border while loading. Possible solutions:
-  - Use CSS `@media not all and (min-resolution:.001dpcm) { img[loading="lazy"] { clip-path: inset(0.5px) } }`
+- [Safari 15+](https://bugs.webkit.org/show_bug.cgi?id=243601) displays a gray border while loading. Possible solutions:
+
+  - Use CSS
+
+  ```
+  /* Safari v10.1 to v15.6 */
+  @media not all and (min-resolution: 0.001dpcm) {
+    img[loading="lazy"] {
+      clip-path: inset(0.5px);
+    }
+  }
+  /* Safari v16.0+ */
```

Comment:
Can we target both safari 15 and 16 with a single media query?

Reply:
using: 
``` 
@media not all and (min-resolution: 0.001dpcm), (min-resolution: 0.001dpcm) {
  @supports (-webkit-appearance: none) and (stroke-color: transparent) {
    img[loading="lazy"] {
      clip-path: inset(0.5px);
    }
  }
}
```
 media query should work but would make the example less clear I think. 
write this doc just to explain to developers, because in fact, developers still need to change it, such as `0.5px` maybe need to change `1px`. 

WDYT?

Reply:
I think 1px will be too much and make the image look smaller in Safari. We want this value to be the smallest possible change to the page while still removing the border.

Reply:
got it! thank you! 
anyway, is a good way to fix this!

Reply:
I changed to 0.6px 👍